### PR TITLE
update robustness meeting time

### PR DESCRIPTION
**Pull request by samuelbartels20**
Please review

### CODE REVIEW

**Comment by samuelbartels20** [comment]
@jmhbnz

File: content/en/community/_index.md [hunk 1]
```diff
@@ -43,11 +42,12 @@ menu:
 {{< page/toc collapsed=true placement="inline" >}}
 </div>
 
+
 <div class="row">
 <div class="col-12 col-lg-8">
 
 {{% alert color="success" %}}
-  <i class='fas fa-users mr-1'></i> Our community values respect and
+  `<i class='fas fa-users mr-1'></i>` Our community values respect and
```
Comment (jmhbnz) [inline content/en/community/_index.md:50]: It looks like this update is not rendering correctly, the deploy preview here shows html tags https://deploy-preview-918--etcd.netlify.app/community/

image

**Review by jmhbnz — COMMENTED** [review]
Thanks for picking this up @samuelbartels20 - Looks close just one issue below we need to address.

File: content/en/community/_index.md (hunk 1 above)
Comment (samuelbartels20) [reply content/en/community/_index.md:50]: Ok. I'm getting it done right away.

File: content/en/community/_index.md [hunk 2]
```diff
@@ -43,11 +42,12 @@ menu:
 {{< page/toc collapsed=true placement="inline" >}}
 </div>
 
+
 <div class="row">
 <div class="col-12 col-lg-8">
 
 {{% alert color="success" %}}
-  <i class='fas fa-users mr-1'></i> Our community values respect and
+  Our community values respect and
```
Comment (spzala) [inline content/en/community/_index.md:50]: Curious why you deleted `<i class='fas fa-users mr-1'></i>`? Thanks! Lgtm otherwise.

**Review by spzala — COMMENTED** [review]
lgtm but have one inline questions. The lint errors seems not related to changes. Thanks @samuelbartels20

**Pull request closed without merging**